package.json

```diff
@@ -29,6 +29,7 @@
     "react-leaflet": "^4.2.1",
     "react-modal": "^3.16.1",
     "react-responsive-carousel": "^3.2.23",
+    "react-router-dom": "^6.26.1",
```

Comment:
Do you need this library?

If not you can uninstall it by typing `npm uninstall react-router-dom` to your terminal